src/sysupdate/sysupdate-transfer.h

```diff
@@ -44,9 +45,13 @@ struct Transfer {
         /* If we write to a partition in a partition table, the metrics of it */
         PartitionInfo partition_info;
         PartitionChange partition_change;
+
+        Context *ctx;
```

Comment:
I'd call this field "context".

We are happy to abbreviate names of local scope inside functions, where you don't have to track the name over broader pieces of codes. But for struct members we typically avoid that, and spell out the names.